cmd/util/cmd/execution-state-extract/execution_state_extract_test.go

```diff
@@ -108,7 +114,7 @@ func TestExtractExecutionState(t *testing.T) {
 				require.NoError(t, err)
 
 				stateCommitment, _, err = f.Set(update)
-				//stateCommitment, err = f.UpdateRegisters(keys, values, stateCommitment)
+				// stateCommitment, err = f.UpdateRegisters(keys, values, stateCommitment)
```

Comment:
Should this and the other commented-out lines below maybe just be removed?

Reply:
I don't know why it was commented out instead of being removed.

We can address this and similar existing issues in a separate cleanup PR which may require contacting other people.